packages/consumption/src/modules/requests/itemProcessors/proposeAttribute/ProposeAttributeRequestItemProcessor.ts

```diff
@@ -84,12 +84,14 @@ export class ProposeAttributeRequestItemProcessor extends GenericRequestItemProc
             return commonQueryValidationResult;
         }
 
-        if (requestItem.query instanceof RelationshipAttributeQuery && requestItem.query.owner.toString() !== "") {
-            return ValidationResult.error(
-                ConsumptionCoreErrors.requests.invalidRequestItem(
-                    "The owner of the given `query` can only be an empty string. This is because you can only propose Attributes where the Recipient of the Request is the owner anyway. And in order to avoid mistakes, the owner will be automatically filled for you."
-                )
-            );
+        if (requestItem.query instanceof RelationshipAttributeQuery) {
+            if (requestItem.query.owner.toString() !== "") {
+                return ValidationResult.error(
+                    ConsumptionCoreErrors.requests.invalidRequestItem(
+                        "The owner of the given `query` can only be an empty string. This is because you can only propose Attributes where the Recipient of the Request is the owner anyway. And in order to avoid mistakes, the owner will be automatically filled for you."
+                    )
+                );
+            }
```

Comment:
Why did you change this?

Reply:
That was not intentional. I had done a bit of refactoring based on another review comment, which I then undid. I must have forgotten to undo the change you are referring to. :D